src/test/groovy/edu/ie3/datamodel/io/factory/input/participant/HpInputFactoryTest.groovy

```diff
@@ -43,7 +43,7 @@ class HpInputFactoryTest extends Specification implements FactoryTestHelper {
 
 		when:
 		Optional<HpInput> input = inputFactory.getEntity(
-				new HpInputEntityData(parameter, operatorInput, nodeInput, typeInput, thermalBusInput))
+				new HpInputEntityData(parameter,operatorInput, nodeInput, typeInput, thermalBusInput))
```

Comment:
:(

Reply:
@sensarmad don't get this - would you mind clarifying pls?!